app/Actions/Albums/Top.php

```diff
@@ -35,7 +35,7 @@ public function __construct()
 	 * Note: the array may include password-protected albums that are not
 	 * accessible (but are visible).
 	 *
-	 * @return array[Collection[Album]]
+	 * @return array[Collection<Album>]
```

Comment:
`array<Collection<Album>>`

Reply:
Have you ever tried the "insert a suggestion" of Github ?

Reply:
Honestly, I did not know that this feature exists. :see_no_evil: I will use it from now on.